### PR TITLE
feat(Videoland): buttons, cover & privacy settings

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Added buttons, cover & privacy settings & refactor of the old code

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Review by EncryptedDev — APPROVED** [review]

**Comment by EncryptedDev** [comment]
@Bas950 also check for lang strings

File: websites/V/Videoland/presence.ts [hunk 1]
```diff
@@ -21,60 +21,126 @@ async function fetchShowTitle() {
...
+				presenceData.buttons = [
+					{
+						label: "Bekijk Series",
+						url: href,
+					},
+				];
+				break;
+			}
+			case "tv-programmas": {
+				presenceData.details = "Bekijkt Tv Programma's";
```
Comment (Bas950) [inline websites/V/Videoland/presence.ts:93]: Doe gewoon de t en p in kleine letters

File: websites/V/Videoland/presence.ts [hunk 2]
```diff
@@ -21,60 +21,126 @@ async function fetchShowTitle() {
...
+				presenceData.buttons = [
+					{
+						label: "Bekijk Tv Programma's",
```
Comment (Bas950) [inline websites/V/Videoland/presence.ts:96]: Hier ook

File: websites/V/Videoland/presence.ts [hunk 3]
```diff
@@ -21,60 +21,126 @@ async function fetchShowTitle() {
...
+					? "Gepauzeerd"
+					: "Aan het afspelen";
+				if (!video.paused) {
+					presenceData.endTimestamp = presence.getTimestampsfromMedia(
+						document.querySelector("video")
+					)[1];
+				}
+				presenceData.buttons = [
+					{
+						label: "Bekijk Video",
```
Comment (Bas950) [inline websites/V/Videoland/presence.ts:132]: Kleine v

File: websites/V/Videoland/presence.ts [hunk 4]
```diff
@@ -21,60 +21,126 @@ async function fetchShowTitle() {
...
+						url: href,
+					},
+				];
+				break;
+			}
+			case "kids": {
+				presenceData.details = "Bekijkt alle kids series";
+				presenceData.buttons = [
+					{
+						label: "Bekijk Series",
```
Comment (Bas950) [inline websites/V/Videoland/presence.ts:86]: Kleine s

File: websites/V/Videoland/presence.ts [hunk 5]
```diff
@@ -21,60 +21,126 @@ async function fetchShowTitle() {
...
+			case "series": {
+				if (meta !== "Videoland") presenceData.details = `Bekijkt ${meta}`;
+				else presenceData.details = "Bekijkt alle series";
+				presenceData.largeImageKey =
+					document
+						.querySelector('[class="detail-meta-cover__image"]')
+						?.getAttribute("src") ?? "https://i.imgur.com/DbrnNCZ.gif";
+				presenceData.buttons = [
+					{
+						label: "Bekijk Serie",
```
Comment (Bas950) [inline websites/V/Videoland/presence.ts:76]: Hier ook kleine s

File: websites/V/Videoland/presence.ts [hunk 6]
```diff
@@ -21,60 +21,126 @@ async function fetchShowTitle() {
...
+			case "films": {
+				if (meta !== "Videoland") presenceData.details = `Bekijkt ${meta}`;
+				else presenceData.details = "Bekijkt alle films";
+				presenceData.largeImageKey =
+					document
+						.querySelector('[class="detail-meta-cover__image"]')
+						?.getAttribute("src") ?? "https://i.imgur.com/DbrnNCZ.gif";
+				presenceData.buttons = [
+					{
+						label: "Bekijk Film",
```
Comment (Bas950) [inline websites/V/Videoland/presence.ts:61]: Kleine f

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/V/Videoland/presence.ts (hunk 3 above)
Comment (DarkVillager) [reply websites/V/Videoland/presence.ts:132]: Nee title case?

File: websites/V/Videoland/presence.ts (hunk 4 above)
Comment (DarkVillager) [reply websites/V/Videoland/presence.ts:86]: Title case?

File: websites/V/Videoland/presence.ts (hunk 5 above)
Comment (DarkVillager) [reply websites/V/Videoland/presence.ts:76]: Title case?

File: websites/V/Videoland/presence.ts (hunk 6 above)
Comment (DarkVillager) [reply websites/V/Videoland/presence.ts:61]: Title case?

File: websites/V/Videoland/presence.ts (hunk 2 above)
Comment (DarkVillager) [reply websites/V/Videoland/presence.ts:96]: Title case?

**Review by Bas950 — APPROVED** [review]

**Pull request merged**